doc/lightningd-config.5

```diff
@@ -418,7 +424,7 @@ to gain our eternal gratitude!
 
 .SH AUTHOR
 
-Rusty Russell &lt;\fIrusty@rustcorp.com.au\fR&gt; wrote this man page, and
+Rusty Russell &lt;\fBNone\fR (\fIrusty@rustcorp.com.au\fR)&gt; wrote this man page, and
```

Comment:
This kind of gratuitous format change seems to keep happening: do we need to fixup the md files somehow?

Reply:
Yep, it seems to be an issue with `mrkd` which assumes we have link text even on bare links. We can fix this in `mrkd` since @darosior has a fork, either by ignoring non-marked up links of default them to the link URL.

Reply:
PR up: https://github.com/darosior/mrkd/pull/1

Will switch the requirements.txt file to use that branch once the PR gets merged.

Reply:
I've also PRed it upstream : https://github.com/refi64/mrkd/pull/2

Reply:
Nice, hadn' noticed that refi64 is still merging PRs :-)